tools/dev/msftidy.rb

```diff
@@ -38,7 +41,91 @@ def cyan
   end
 end
 
-class Msftidy
+class RuboCopRunnerException < StandardError; end
+
+# Wrapper around RuboCop that requires modules to be linted
+# In the future this class may have the responsibility of ensuring core library files are linted
+class RuboCopRunner
+
+  ##
+  # Run Rubocop on the given file
+  #
+  # @param [String] full_filepath
+  # @param [Hash] options specifying autocorrect functionality
+  # @return [Integer] RubCop::CLI status code
```

Comment:
I think this meant to be `RuboCop::CLI` here vs `RubCop::CLI`?
```suggestion
  # @return [Integer] RuboCop::CLI status code
```